package.json

```diff
@@ -23,9 +23,15 @@
   "dependencies": {
     "ethereumjs-abi": "^0.6.2",
     "ethereumjs-icap": "^0.3.1",
-    "ethereumjs-tx": "^1.1.1",
-    "ethereumjs-units": "^0.1.0",
```

Comment:
Why remove units?

Reply:
``ethereumjs-all`` is just too hard to update, felt to me that it's just not realistic to keep on going with this.

I combined the ``wallet`` bundels into one, wasn't sure about that though. Is this such a burden to have this in one bundle?

Reply:
Yep, see above. The main reason:
- most use cases only require the core functionality, which is about 5-6k of code
- some other users do require BIP32 (HD keys), that draws in another 5-6k worth of code
- minuscule amount of users would want to import from obsolete wallets, those need wallet-thirdparty, because that imports a lot (don't remember but could be >100k of code)

Reply:
It's not hard, it takes time, so someone has to do it. What makes you assume, that this will be done from now on every two weeks, when before it hasn't been done for over a year?

I'll improve on what I have and re-add the different wallet versions with the next update.

Reply:
I also didn't include ``ethereumjs-all`` because it didn't fit in the versioned-logic of the other builds, and I didn't like that. Maybe I can instead add the date like ``ethereumjs-all-2017-10-27``, then this is also transparent to users if the build is outdated or rather current.

Reply:
I don't actually think this must be done every two weeks, rather every time a substantial release is made. Actually the first substantial release was only made in the last 2 months to the VM, before that nothing has happened for a year.

Yeah, dated builds (or hash (first 8 nibbles) of say `package.json`?) are definitely a good idea.